cantools/database/can/signal.py

```diff
@@ -387,6 +389,19 @@ def multiplexer_signal(self):
     def multiplexer_signal(self, value):
         self._multiplexer_signal = value
 
+    @property
+    def spn(self):
+        """The SPN (Suspect Parameter Number) value if the signal
+        has this attribute, ``None`` otherwise.
```

Comment:
You should mention J1939.

Reply:
You may that I need to validate that DBC file is based on J1939 before SPN parsing?

Reply:
Nothing like that. Just Mention J1939 in the docstring. SPN is part of the J1939 specification.

Reply:
Got it, mentioned.